tests/test_resources/test_clusters/test_on_demand_cluster.py

```diff
@@ -299,13 +331,20 @@ def test_logs_surfacing_scheduler_basic_flow(self, cluster):
         ]  # clean log formatting
         cluster_logs = "\n".join(cluster_logs)  # make logs list into one string
 
+        resp_data = get_logs_data_resp.json().get("data")
+        # check that the status was updated in the DB in the DEFAULT_STATUS_CHECK_INTERVAL minute(s),
+        # cause the status periodic check runs every DEFAULT_STATUS_CHECK_INTERVAL minute(s).
+        status_updated_at = datetime.datetime.fromisoformat(
```

Comment:
simimlar note as above, this shouldn't really have to depend on den to test that logs collection is working.. maybe we just check the config for line numbers to ocnfirm that they've been updated?

Reply:
Not sure that checking if the line numbers has been updated is the best solution here:

1. There is an edge case where the lines are not updated, because no new logs were generated (if the cluster is just up, and not functions / modules are running)
2. We need to wait at least 60 seconds to test this, since the cluster checks loop is updating the line numbers, and we need to wait that it will be executed at least once during the test... This will add sufficient latency to the test suite.